dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'VG': [Ilhas Virgens Britânicas, Virginense*, Road Town]
+    'VI': [Ilhas Virgens Americanas, Ilhas Virgens dos Estados Unidos, Virginense*, Charlotte Amalie, Carlota Amália]
+
+  CENTER:
+    'BZ': [Belize, Belizenh*, Belizense*, Belmopan, Belmopã]
+    'CR': [Costa Rica, Costa-Riquenh*, Costa-Riquense*, Costarriquense*, Costa-Ricense*, Costarricense*, Ticos, San José, São José]
+    'GT': [Guatemala, Guatemaltec*, Guatemalense*, Guatemalês*, Guatemales*, Cidade da Guatemala]
+    'HN': [Honduras, Hondurenh*, Tegucigalpa]
+    'MX': [México, Estados Unidos Mexicanos, Mexican*, Cidade do México]
+    'NI': [Nicarágua, Nicaraguense*, Nica, Manágua]
+    'PA': [Panamá, Panamenh*, Panamense*, Cidade do Panamá]
+    'SV': [El Salvador, Salvadorenh*, San Salvador, São Salvador]
+
+  SOUTH:
+    'AR': [Argentina, Argentin*, Buenos Aires]
+    'BO': [Bolívia, Bolivian*, Sucre, La Paz]
+    'BR': [Brasil, Brasileir*, Brasília, São Paulo, Rio de Janeiro]
+    'CL': [Chile, Chilen*, Santiago, Santiago do Chile]
+    'CO': [Colômbia, Colombian*, Bogotá]
+    'EC': [Equador, Equatorian*, Quito]
+    'FK': [Ilhas Malvinas, Ilhas Falkland, Falklands, Falklander*, Malvinense*]
+    'GF': [Guiana Francesa, Franco-Guianense*]
+    'GY': [Guiana, Guyana, Guianês*, Guianense*]
+    'PE': [Peru, Peruan*, Peruvian*, Lima]
+    'PY': [Paraguai, Paraguai*, Paraguaian*, Assunção]
+    'SR': [Suriname, Surinamense*, Surinamês*, Paramaribo]
+    'UY': [Uruguai, Uruguai*, Montevidéu, Montevideu]
+    'VE': [Venezuela, Venezuelan*, Caracas]
+
+  NORTH:
+    'BM': [Bermudas, Bermudenh*, Bermudense*]
+    'CA': [Canadá, Canadense*, Canadian*, Ottawa, Otava, Toronto, Quebec, Cidade de Quebec]
+    'GL': [Gronelândia, Groenlândia, Groelândia, Gronelandês*, Groenlandês, Nuuk, Nuque]
+    'PM': [São Pedro e Miquelão, São Pedro e Miquelon, Saint-Pierre e Miquelon, Miquelonês*, São-Pedrense*, São Pedro]
+    'US': [Estados Unidos, Estados Unidos da América, América, EUA, Washington, Nova Iorque, Nova York]
+
+ASIA:
+  CENTER:
+    'KG': [Quirguistão, Quirguizistão, Quirguízia, República Quirguiz, Quirguiz*, Quirguistanês*, Bisqueque]
+    'KZ': [Cazaquistão, Cazaque*, Cazaquistanês*, Casac*, Nur-Sultã, Nur Sultã]
+    'TJ': [Tajiquistão, Tadjiquistão, Tajique*, Tadjique*, Tajiquistanês*, Tadjiquistanês*, Duxambé, Duchambé]
+    'TM': [Turquemenistão, Turcomenistão, Turquemen*, Turcomen*, Turquemenistanês*, Asgabate, Asgabade, Ascabade]
+    'UZ': [Uzbequistão, Usbequistão, Usbeque*, Uzbeque*, Usbequistanês*, Tasquente, Tashkent]
+
+  EAST:
+    'CN': [China, RPC, Chines*, Chinês*, Pequim, Xangai, Shanghai]
+    'HK': [Hong Kong, Honconguês, Honcongues*]
+    'JP': [Japão, Japonês*, Japones*, Japonense*, Japônic*, Japónic*, Nipônic*, Nipónic*, Tóquio]
+    'KP': [Coreia do Norte, Coréia do Norte, República Popular Democrática da Coreia, RPDC, Norte-Corean*, Pyongyang, Pionguiangue]
+    'KR': [Coreia do Sul, Coréia do Sul, República da Coreia, Coreia, Sul-Corean*, Seul]
```

Comment:
Coreia will match all entries for KP too, maybe remove?

Reply:
Removed